lib/lita/handlers/versioner.rb

```diff
@@ -109,14 +109,19 @@ def github_handler(request, response)
         when "pull_request"
           # https://developer.github.com/v3/activity/events/types/#pullrequestevent
           # If the pull request is merged with some commits
-          if payload["action"] == "closed" && payload["pull_request"]["merged"]
-            new_version = bump_version_in_git
-            trigger_build("v#{new_version}")
+          if payload["action"] == "closed"
+            if payload["pull_request"]["merged"]
+              inform("'#{payload["pull_request"]["html_url"]}' is just merged. Working on it...")
```

Comment:
I find working on it helpful as a feedback mechanism that bot has received something. We have this in the `promote` command where when there is no response for a bit people start thinking like "did that go through", "did I do something wrong". So I would like to keep this one since two consecutives ones here will come pretty close to each other.